docs/package.json

```diff
@@ -16,7 +16,8 @@
     "postbuild:previews": "node --require esbuild-register ./scripts/generate-previews.ts",
     "postbuild:sitemap": "node --require esbuild-register ./scripts/generate-sitemap-robotstxt.ts",
     "start": "next start",
-    "test": "jest"
+    "test": "jest",
+    "test:links": "node --require esbuild-register ./scripts/link-checker.ts 0 50 && node --require esbuild-register ./scripts/link-checker.ts 50 100 && node --require esbuild-register ./scripts/link-checker.ts 100"
```

Comment:
Do we really need to split up the links or could we catch the error and continue? 
https://stackoverflow.com/questions/16995184/nodejs-what-does-socket-hang-up-actually-mean